tutorial/part1/chapter4.md

```diff
@@ -259,6 +255,6 @@ if __name__ == '__main__':
 
 ## Summary
 
-Congratulations! You are now able to build a simple app with variuos components, structure them within the app and understand the neccessity of component ids and properties. You have also learned about the concept of app callbacks and how they upgrade your app by adding interactivity. No matter if only one or multiple callbacks, you'll be able to manage your app linking different components together. 
+Congratulations! You are now able to build a simple app with various components, structure them within the app and understand the necessity of component ids and properties. You have also learned about the concept of app callbacks and how they upgrade your app by adding interactivity. No matter if only one or multiple callbacks, you'll be able to manage your app linking different components together. 
 
 Now it's your turn, try yourself to link other components you already know and build your own first interactive app.
```

Comment:
Now it's your turn, try to link other components you learned from chapter 3 and build your own interactive app.

In the next chapter we will learn how to deploy our app to the web.